pkg/intermediate/types.go

```diff
@@ -33,11 +33,17 @@ type AggregationFlowRecord struct {
 	// inter-node flow and record from the node for the case of intra-node flow.
 	ReadyToSend               bool
 	waitForReadyToSendRetries int
-	// isMetaDataFilled is an indicator for IPFIX Mediator to check whether K8s
+	// areCorrelatedFieldsFilled is an indicator for IPFIX Mediator to check whether K8s
 	// metadata are filled for flow record. It is always true for Intra-Node
 	// and ToExternal flows and only applicable for Inter-Node flows that are
 	// not required to be correlated. (e.g. flows with Egress deny rule applied)
-	isMetaDataFilled bool
+	areCorrelatedFieldsFilled bool
+	// Some fields like Pod labels will be filled externally on the IPFIX Mediator,
```

Comment:
"Some fields could be filled externally on aggregation records once before exporting them in IPFIX mediator, e.g., metadata of a flow. areExternalFieldsFilled is an indicator for IPFIX Mediator to check whether
these fields has been filled or not before exporting. ....."
Prefer not to have Pod Labels.

Reply:
Thanks, addressed.